src/Datadog.Trace/SpanContextPropagator.cs

```diff
@@ -46,7 +46,7 @@ public void Inject<T>(SpanContext context, T headers)
             // lock sampling priority when span propagates.
             context.TraceContext?.LockSamplingPriority();
```

Comment:
The SpanContextPropagator might also need to be feature-flagged based on the desired traceId implementation, because Datadog writes the traceId as a decimal number, whereas this is currently writing the traceId as a hexadecimal number

Reply:
This feature flag could also control the Extract method as well

Reply:
Changed that to use decimal for 64bit TraceId, with method to parse being chosen in Convention class.